@xen-orchestra/web-core/lib/types/donut-chart.type.ts

```diff
@@ -0,0 +1,5 @@
+export type DonutColor = 'success' | 'warning' | 'error' | 'unknown'
+export type DonutSegment = {
+  value: number
+  color: DonutColor
+}
```

Comment:
`DonutSegmentColor` would probably be a more suitable name.

```suggestion
export type DonutSegmentColor = 'success' | 'warning' | 'error' | 'unknown'

export type DonutSegment = {
  value: number
  color: DonutSegmentColor
}
```